tests/contrib/test_get_footer.py

```diff
@@ -0,0 +1,66 @@
+from django.conf import settings
+from django.core.cache import cache
+from django.core.management import call_command
+from django.test import override_settings
+
+from meinberlin.apps.contrib.tasks import periodic_footer_update
+from meinberlin.apps.contrib.templatetags.contrib_tags import get_external_footer
+
+
+def test_get_footer(clear_cache_footer):
+    footer = cache.get("footer")
+    assert footer is None
+    footer = get_external_footer()
```

Comment:
these tests will fail if the berlin.de servers are not available, which happened more than one would think and sometimes for multiple days. I'm not sure if it's a good idea to rely on them.  I guess we could use something like `request-mocks` or just keep the tests which don't download anything. What do you think?

Reply:
I will apply a mocked request